### PR TITLE
Pixi v4

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
pixi-v4 commit. Create new branch, please

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,9 +40,10 @@
     "README.md"
   ],
   "peerDependencies": {
-    "pixi.js": "^3.0.2"
+    "pixi.js": "^4.0.0"
   },
   "dependencies": {
+    "glslify": "^5.0.2",
     "brfs": "^1.4.1"
```
Comment (englercj) [inline package.json:47]: is brfs needed anymore after these are changed to glslify?

Comment (ivanpopelyshev) [inline package.json:47]: I dont know what is brfs, I just added glslify

Comment (englercj) [inline package.json:47]: `brfs` is a transform that converts `fs.readFileSync()` calls into the text they would've read in a node environment. I would remove it in favor of glslify as we did in pixi.

**Comment by GoodBoyDigital** [comment]
Looks good to me! Thanks @ivanpopelyshev 👍

**Pull request merged**